### DISCUSSION TITLE
Is it possible to define the build folder?

### DISCUSSION THREAD

**Discussion by victorlmneves** [histoire-dev/histoire#186] [Q&A] [answered]
On the build task, I have set a folder path to store the files but looks like Histoire doesn't accept it as it generates to `.histoire` folder instead of the one I set on the task

```
"story:build": "VITE_MODE=production histoire build --output-dir dist/histoire-static"
```

Is there any other way to achieve this?

Thanks

**Comment by Akryum** [comment, marked as answer]
https://histoire.dev/reference/config.html#outdir

**Reply by victorlmneves** [reply]
I went to the docs but for some reason, I didn't see it :/
Thanks
